src/ComparedQueryable/ComparedQueryable.csproj

```diff
@@ -40,15 +40,22 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="NativeQueryable\CachedReflection.cs" />
+    <Compile Include="EnumerableExecutor[TCollectionElement,TComparison].cs" />

```

Comment:
I wanted a way to include generic classes as their own files. There was a naming convention @ [Convention for Filenames of Generic Classes](http://stackoverflow.com/a/1179581/684271) that I took a liking to here.